napari/_qt/qt_viewer.py

```diff
@@ -418,6 +418,11 @@ def closeEvent(self, event):
             self.pool.clear()
         event.accept()
 
+    def shutdown(self):
```

Comment:
LOL I love that there’s “clear”, “close”, and “shutdown” inside here. Lots to choose. =D 

FWIW I think you picked the right one.

I wonder if it’s necessary to put this in __del__() also? Or will the below methods be called automatically?

Reply:
ooo good question, I don't think it would automatically happen. Should I add a __del__() override?